manuscripts/esquery.py

```diff
@@ -64,216 +61,148 @@ def __get_query_filters(cls, filters=None, inverse=False):
                 # A direct filter and inverse mode
                 continue
             field_name = name[1:] if name[0] == '*' else name
-            query_filters += """
-                {
-                  "match": {
-                    "%s": {
-                      "query": "%s",
-                      "type": "phrase"
-                    }
-                  }
-                }
-            """ % (field_name, filters[name])
-            query_filters += ","
 
-        query_filters = query_filters[:-1]  # Remove the last comma
+            params = {field_name: filters[name]}
+            # trying to use elasticsearch_dsl only and not creating hard coded queries
+            query_filters.append(Q('match_phrase', **params))
 
         return query_filters
 
     @classmethod
     def __get_query_range(cls, date_field, start=None, end=None):
         """
-        Create a string with range DSL query for the date in date_field from start to end dates.
+        Create a filter dict with date_field from start to end dates.
 
         :param date_field: field with the date value
```

Comment:
For dates, and other "complex" data types, please specify the actual format, if possible. For example, in this case, is it "datetime.datetime"? (besides: if it is not something such as datetime.datetime, I would rather consider changing the profile to really use something like that, so that dates are converted into a pythonic format before this method (and others below) are called. We can build some auxiliary function for that, if needed.